src/components/FeatureViews/Accounts/TabComponents/LoadingCourseError.js

```diff
@@ -0,0 +1,13 @@
+import { connect, useSelector } from 'react-redux';
+import React from 'react';
+import loadingCoursesError from "./loadingCoursesError.png";
+
+function LoadingCoursesError(props){
+    return(
+    <div>
+        <img style={{width:"10%"}} src={loadingCoursesError}/>
+        <h2>There was an error loading your {props.error}.</h2>
+    </div>);
+}
+
+export default LoadingCoursesError
```

Comment:
If you want to require the user to have the error prop filled out, be sure to use React.Proptypes. Refer to PriceQuoteForm.js line 396 for an example (and be sure to import React.Proptypes!)